ArduPlane/Attitude.cpp

```diff
@@ -1041,6 +1045,42 @@ void Plane::set_servos(void)
         channel_output_mixer(g.vtail_output, channel_pitch->radio_out, channel_rudder->radio_out);
     } else if (g.elevon_output != MIXING_DISABLED) {
         channel_output_mixer(g.elevon_output, channel_pitch->radio_out, channel_roll->radio_out);
+              //if (both) differential spoilers setup then apply rudder
+              // control into splitting the two elevons on the side of
+              // the aircraft where we want to induce additional drag:
+        if (RC_Channel_aux::function_assigned(RC_Channel_aux::k_dspoiler1) &&
+                   RC_Channel_aux::function_assigned(RC_Channel_aux::k_dspoiler2)) {
+            int16_t ch3 = channel_roll->radio_out;    //diff spoiler 1
+            int16_t ch4 = channel_pitch->radio_out;   //diff spoiler 2
+                   //convert rudder-servo output (-4500 to 4500) to PWM
+                   // offset value and multiply by DSPOILR_RUD_RATE/100:
+            int16_t ruddVal = (int16_t)((int32_t)(channel_rudder->servo_out) *
+                                         g.dspoiler_rud_rate / (SERVO_MAX/5));
```

Comment:
It's a reduction of the formula:  rudder->servo_out * 500 / SERVO_MAX * dspoiler_rud_rate/100.  I've added this to the comment.